config_example.sh

```diff
@@ -708,3 +708,9 @@ set -x
 # This test case is only supported when IP_STACK=v4.
 #
 # export AGENT_TEST_CASES='bad_dns'
+
+# Sets the install-config.yaml's platform type.
+# When undefined, 'baremetal' is assumed. This is the default.
```

Comment:
Would it make sense to set the default value to 'baremetal' then?

Reply:
Yes, that would make sense.